lib/rage/router/backend.rb

```diff
@@ -100,6 +100,14 @@ def lookup(env)
     find(env, constraints)
   end
 
+  def set_params_wrappers(controller = RageController::API)
+    controller.__set_params_wrappers
+
+    controller.subclasses.each do |subclass|
+      set_params_wrappers(subclass)
+    end
+  end
```

Comment:
This part recursively initiates parameters wrapping stack for all controllers starting with RageController::API and down by the Controller subclasses tree